srcpkgs/plasma-workspace/template

```diff
@@ -29,6 +29,12 @@ if [ "$CROSS_BUILD" ]; then
 	configure_args+=" -DKF5_HOST_TOOLING=/usr/lib/cmake"
 fi
 
+post_patch() {
+	if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+		patch -Np0 < $FILESDIR/panelspacer-musl.patch
```

Comment:
You are only applying for musl, I think it should also be applied for 32bit targets, per the comment in the patch.

Reply:
Yeah, and I probably should also test if this actually fixes the problem on 32bit targets. Currently trying to figure out, why the newer code breaks qml.